modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -1,15 +1,14 @@
 @file:Suppress("UnusedImports")
```

Comment:
Who is this for? All of them should be used here.

Reply:
Apparently a detekt exception recently introduced in a different PR

Reply:
Not for this PR then.